web/build.gradle.kts

```diff
@@ -52,7 +52,10 @@ tasks {
   }
 
   clean {
+    delete(".node")
     delete("build")
     delete("dist")
+    delete("node_modules")
+    delete("yarn-error.log")
```

Comment:
hi @SteNicholas Thank you for your contribution!
I agree to delete `yarn-error.log`, But if we clean `.node` and `node_modules`, when we rebuild, it will take quite a bit of time to download.
@ch3yne have any suggestions?

Reply:
@ch3yne
Could you please help confirm this one?

Reply:
Deleting the `node_modules` folder is usually necessary. If you're using the `gradle clean` command to free up disk space, I think it's required.

Reply:
@ch3yne, the purpose of this improvement is that free up disk space via `gradle clean`. Therefore, the `node_modules` folder is deleted in clean task of gradle.